worlds/sc2/Options.py

```diff
@@ -630,13 +633,14 @@ class ChallengeLocations(LocationInclusion):
     See also: Excluded Locations, Item Plando (https://archipelago.gg/tutorial/Archipelago/plando/en#item-plando)
     """
     display_name = "Challenge Locations"
+    default = LocationInclusion.option_disabled
 
 
-class OptionalBossLocations(LocationInclusion):
+class MasteryLocations(LocationInclusion):
     """
-    Enables or disables item rewards for defeating optional bosses.
-    An optional boss is any boss that's not required to kill in order to finish the mission successfully.
-    All Brutalisks, Loki, etc. belongs here.
+    Enables or disables item rewards for overcoming especially difficult challenges.
+    These challenges are often based on Mastery achievements and Feats of Strength.
+    Enable these locations to
```

Comment:
All Feats of Strength and Mastery achievements are possible in vanilla, but it may be worth considering adding more power level logic for Terran and Protoss in a future PR.